tests/Skipper/Skipper/Skip/SkipSkipperTest.php

```diff
@@ -14,21 +15,16 @@
 
 final class SkipSkipperTest extends AbstractTestCase
 {
-    private Skipper $skipper;
-
-    protected function setUp(): void
-    {
-        $this->createContainerWithConfigs([__DIR__ . '/config/config.php']);
-        $this->skipper = $this->make(Skipper::class);
-    }
-
+    #[RunInSeparateProcess]
```

Comment:
FYI: `#[RunInSeparateProcess]` had to be added because `testCheckerSkipRulesAreMergedRecursively()` uses different config file, and since `SimpleParameterProvider` is a static parameter bag it was reusing values between tests (configs, especially skip rules, were merged), which was causing tests to fail.